making-a-proposal.md

```diff
@@ -0,0 +1,207 @@
+# Interop Proposers Guide
+
+As set out in the [README](README.md), the goal of the Interop project
+is to improve the web by making it easier to create websites and
+webapps that work well in every browser and browser engine.
+
+This guide lists the signals that the Interop team uses when selecting
+focus areas, in order to help you understand what is likely to be
+accepted, and help you advocate for your proposal in the most
+effective way.
+
+Note that apart from those under Essential Criteria, none of the items
+listed are considered binding, and the different organizations that
+are part of the Interop team may use the criteria in different ways,
+or supplement these with their own specific considerations.
+
+For reference, here are some proposals from past years that are well structured. 
+Please note that the proposal template will have changes year over year. 
+We encourage you to read the guide in its entirety for the latest guidance.
+
+## Submitting a proposal
+
+Submit a proposal using one of the following links:
+
+- [Propose an Interop 2026 Focus
+  Area](https://github.com/web-platform-tests/interop/issues/new?template=focus-area-proposal.yml)
+  – for features that already have high quality specifications and
+  tests, but aren't implemented by all browsers.
+- [Proposal an Interop 2026 Investigation
+  Effort](https://github.com/web-platform-tests/interop/issues/new?investigation-effort-proposal.yml)
+  – for features that are widely implemented, but lack a high quality
```

Comment:
The features may or may not be widely implemented, but generally the problem is that they either lack tests or testing infrastructure, so can't be included as a focus area (lack of a high quality specification would also be a blocker, but it's something that needs to be fixed at the WG level).